app/services/mentions/create_all.rb

```diff
@@ -10,32 +10,55 @@ def self.call(*args)
 
     def call
       # Only works for comments right now.
+      mentioned_users = users_mentioned_in_notifiable_text_that_are_existing_and_not_its_owners
+
+      delete_mentions_removed_from_notifiable_text(mentioned_users)
+      create_mentions_for(mentioned_users)
+    end
+
+    private
+
+    def users_mentioned_in_notifiable_text_that_are_existing_and_not_its_owners
```

Comment:
This method name feels a bit verbose to me; how would you feel about shortening it and perhaps adding a clarifying comment?

Reply:
Hi @vaidehijoshi, thanks for help.

Regarding the long name, I was inspired by [Naming as a process](https://www.digdeeproots.com/articles/on/naming-as-a-process/) series. It's called "a honest name", and the next step would be to replace it with a name that reveals domain intent.

Fortunately, I was suggest a better shorter name.